meta-genivi-dev/recipes-dev-hmi/genivi-dev-platform-hmi/gdp-new-hmi.bb

```diff
@@ -1,5 +1,5 @@
 SRC_URI = "git://github.com/GENIVI/hmi-layout-gdp.git"
-SRCREV = "d0ca9d072012d8ac333537d9b922d9242e022efe"
+SRCREV = "79a4f69e7588a6efc4b4c95abec7794216588087"
```

Comment:
@chbae It's not that it's not enough, it's that it's wrong. You're summarizing a side-effect, not the change. I can't be writing commit messages for everyone. :) Please try yourself first and if you fail, I'll help. Deal? :)